Source/IGListCollectionView.m

```diff
@@ -11,25 +11,6 @@
 
 @implementation IGListCollectionView
 
-- (void)commonInit {
-    self.backgroundColor = [UIColor whiteColor];
-    self.alwaysBounceVertical = YES;
-}
-
-- (instancetype)initWithFrame:(CGRect)frame collectionViewLayout:(UICollectionViewLayout *)layout {
-    if (self = [super initWithFrame:frame collectionViewLayout:layout]) {
-        [self commonInit];
```

Comment:
agree, we should keep `IGListCollectionView` for those warnings.

but, we should consider #240 if more requests come up.